.vib/haproxy/cypress/cypress/integration/haproxy_spec.js

```diff
@@ -0,0 +1,6 @@
+/// <reference types="cypress" />
+
+it('can access backend server', () => {
```

Comment:
It is clear to me why and how you are configuring haproxy with nginx as sidecar because we discussed it, but I fear it could be confusing for users down the road. Could you please add a brief comment on it here?